ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte

```diff
@@ -174,47 +189,70 @@
     <LoadingFlat />
   {:else}
     <!-- else content here -->
-    <div class="option">
-      {#each rulePresets as presetOption, index}
-        <label class="inline-block mr-2">
-          <input type="checkbox"
-            name="presets"
-            value={presetOption.name}
-            id="option{index}"
-            on:input={handlePresetInput}
-            bind:group={presetSelection}
-          >
-          <p class="inline-block align-baseline">{presetOption.name}</p>
+    {#if showHistory}
+      <div class="mb-4 link cursor-pointer" on:click={() => showHistoryLog()} on:keypress>
+        <i class="fas fa-arrow-left"></i>
+        Back
+      </div>
+      {#each historyLog as log}
+        <div class="mb-4 overflow-hidden border">
+          <div class="content-center px-6 py-4">
+            <div class="font-sans font-bold">
+              {new Date(log.timestamp).toLocaleString()}
+            </div>
+            <div>
+              Selected Rules: {log.selectedRules}
```

Comment:
Can we clean this up to add spaces after the commas so it's a bit easier to read?

<img width="503" alt="Screenshot 2023-09-20 at 5 17 32 pm" src="https://github.com/SSWConsulting/SSW.CodeAuditor/assets/11418832/35531d4b-f6c6-4853-87ca-675318e7ae44">